COPIA/src/components/header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+class Header extends React.Component {
+    render() {
+        return (
+            <div>
+                <nav className='navbar navbar-light bg-secondary'>
+                    <a className='navbar-brand text-white' href='https://getbootstrap.com/docs/4.3/components/navbar/'>
```

Comment:
remember the first requirement of our application --> 
Our "board" or blackboard must have an **editable**  title.